packages/apidom-parser-adapter-json-schema-yaml-2020-12/package.json

```
@@ -0,0 +1,63 @@
+{
+  "name": "@swagger-api/apidom-parser-adapter-json-schema-yaml-2020-12",
+  "version": "1.0.0-beta.7",
```

This will go for beta.8 release, after JSON parser adapter

```suggestion
  "version": "1.0.0-beta.8",
```